packages/devtools_app/lib/src/shared/preferences/_inspector_preferences.dart

```diff
@@ -81,11 +84,11 @@ class InspectorPreferencesController extends DisposableController
     );
   }
 
-  Future<void> _initInspectorV2Enabled() async {
-    await _updateInspectorV2Enabled();
+  Future<void> _initLegacyInspectorEnabled() async {
+    await _updateLegacyInspectorEnabled();
```

Comment:
should remove the old 'inspector.inspectorV2Enabled' key as part of initializing the legacy enabled key?

Reply:
I've opened up a bug for that: https://github.com/flutter/devtools/issues/8667

We need to add functionality to remove storage keys, which I would like to do in a separate PR. For now, this means that `inspector.inspectorV2Enabled` will be stored in the browser storage, but we aren't reading it so it shouldn't have any impact.